web/src/routes/auth/change-password/+page.svelte

```diff
@@ -18,5 +19,12 @@
     enter the new password below.
   </p>
 
-  <ChangePasswordForm user={$user} on:success={() => goto(AppRoute.AUTH_LOGIN)} />
+  <ChangePasswordForm
+    user={$user}
+    on:success={async () => {
+      await goto(AppRoute.AUTH_LOGIN);
+      resetSavedUser();
+      await logout();
+    }}
```

Comment:
Can you move this on a handleSuccess method defined in the `<script>` section of the file instead?

Reply:
done. I called it `onSuccess` though. That way when we migrate from event dispatchers to callback props for Svelte 5 we can use the shorthand syntax of `{onSuccess}` rather than having to write `onSuccess={handleSuccess}`